README.md

```diff
@@ -20,6 +20,11 @@ where we cannot use systems like [debezium](http://debezium.io) or
 have access to the WAL logs and cannot install native extensions or run binaries
 on the database host machine.
 
+The following statements are used to send updates to the relevent topic:
+
+* `INSERT`
+* `UPDATE`
+* `DELETE`
```

Comment:
We also have a `SNAPSHOT` event

Reply:
ah right, these aren't really event types, but SQL operations. Maybe I should improve on this a bit.